src/metabase/analytics/stats.clj

```diff
@@ -873,6 +888,16 @@
                 (update-in [:stats :stats :cache] dissoc :num_queries_cached_unbinned))
      :snowplow-stats (snowplow-anonymous-usage-stats stats)}))
 
+(defn- deep-string-keywords
+  "Snowplow data will not work if you pass in keywords, but this will let use use keywords all over."
+  [data]
+  (walk/postwalk
+   (fn [x] (if (keyword? x) (-> x u/->snake_case_en name) x))
+   data))
```

Comment:
I wanted to put this inside of snowplow/track-event!, however that is used in a lot of places and I don't have great visibility into errors caused by subtle changes in the stats ping data.